openassessment/xblock/staff_area_mixin.py

```diff
@@ -495,3 +522,39 @@ def _cancel_workflow(self, submission_uuid, comments, requesting_user_id=None):
             msg = ex.message
             logger.exception(msg)
             return {"success": False, 'msg': msg}
+
+    def _return_workflow(self, submission_uuid, comments, requesting_user_id=None):
```

Comment:
this method is very similar to _cancel_workflow. Are you thinking about some kind of generalization?

Reply:
I think methods should be separated. because if in the future we would change return flow we can do it faster. It will be more flexible.

Reply:
Ok